ruby/rails6-shakapacker/app/config/appsignal.yml

```
@@ -0,0 +1,42 @@
+default: &defaults
+  # Your push api key, it is possible to set this dynamically using ERB:
+  # push_api_key: "<%= ENV['APPSIGNAL_PUSH_API_KEY'] %>"
+
+  revision: "a000016"
```

For the front-end test setups, since we usually use them to test sourcemap behaviour, I recently changed it so that [a new "fake" revision is generated automatically if one isn't manually provided](https://github.com/appsignal/frontend-test-setups/commit/0f9fb9bc16fb565949db1b62c2528be4155f747f#diff-455425ccbc4ec215dc158c03b7fee85c192ab0372ab283cabe7e9d8f5f363369R60-R62). This spares one from having to manually change it between reboots.

Maybe that would be useful behaviour here as well, since the purpose of this test setup is also testing sourcemaps? If so, because this YAML is an ERB template, we could do the exact same thing (and remove the notice from the README about changing it manually):

```suggestion
  revision: "<%= "0123456789abcdef".chars.sample(7).join + "-demo" %>"
```

---

It can't be a new revision every time then the generated sourcemap is uploaded with a different revision than the one used by the app it will not link the sourcemap to the error backtrace. If this file is changed rails is restarted and the sourcemap is reuploaded so it shouldn't be too much trouble.